source/Calamari.Common/Features/Behaviours/RollbackScriptBehaviour.cs

```diff
@@ -21,10 +21,7 @@ public class RollbackScriptBehaviour : PackagedScriptRunner, IBehaviour
 
         public bool IsEnabled(RunningDeployment context)
         {
-            var features = context.Variables.GetStrings(KnownVariables.Package.EnabledFeatures)
-                                  .Where(s => !string.IsNullOrWhiteSpace(s)).ToList();
-
-            return features.Contains(KnownVariables.Features.CustomScripts);
+            return true;
```

Comment:
Why doesn't this depend on the feature?

Reply:
So I went back and reviewed how this rollback is done in https://github.com/OctopusDeploy/Calamari/blob/master/source/Calamari/Deployment/Conventions/RollbackScriptConvention.cs and it seems there is no predicate for it